http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4031,6 +4031,11 @@ components:
         sourceWorkspaceId:
           type: string
           description: Optional id of source workspace if app is a clone.
+        autoDeleteThresholdDuration:
+          type: string
+          description: The duration of idle time to elapse before the app is
```

Comment:
shall we mention "since last time the app is accessed by user". Also mention that `Inf` means indefinite, which means no auto delete